assn2_AioHTTP/web_app_aiohttp.py

```diff
@@ -0,0 +1,40 @@
+import zipfile
+from aiohttp import web
+
+
+async def index():
+    return web.Response(text=
+    """
+    <!DOCTYPE html>
```

Comment:
You have an html file for this, use it to render as response instead of writing html script as text here